lib/cfn-nag/custom_rules/EC2NetworkAclEntryPortRangeRule.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class EC2NetworkAclEntryPortRangeRule < BaseRule
+  def rule_text
+    'EC2 NetworkACL Entry Port Range must be defined for TCP/UDP protocols and should probably not allow all ports'
```

Comment:
let's change this verbiage to focus on the port range as opposed to the syntactic rules.  perhaps:

TCP/UDP protocol NetworkACL entries possibly should not allow all ports.